src/configurations/adknowledgeportal/resources.ts

```diff
@@ -1,7 +1,9 @@
 export const computationalSql = 'Select * from syn20337467'
 export const toolSql = 'select * from syn22219805'
 export const dataSql = 'SELECT * FROM syn11346063.12'
+export const dataOnStudiesPageSql = "SELECT id, metadataType, dataType, assay FROM syn11346063.12 WHERE `resourceType` = 'metadata'"
```

Comment:
Moved sql into resources.ts so that stakeholders can bump the Synapse View version here after they have a data release.

Reply:
Good idea!